### DISCUSSION TITLE
Boot/shutdown

### DISCUSSION THREAD

**Discussion by cumafo** [PiSupply/PiJuice#790] [Q&A]
Using a pijuice on a pi3, is it possible to always have power on the hat but boot / shutdown following power on the pi3?

No need for manual boot/shutdown.

**Comment by tvoverbeek** [comment]
Do you have both the PiJuice Hat and the Pi3 connected to a power supply (= using 2 power supplies)?
If so, you need a user script regularly polling the Pi3 power input and shutdown the Pi3 when the power input disappears.
Make sure you also switch off the 5V power from the PiJuice to the Pi GPIO.
Then the Pi3 will boot when it is reconnected to external power.

**Comment by cumafo** [comment]
Thanks!
The application is used in a boat for surveillance (camera/vpn/mail).
The pi & hat have the same power source (lead battery).
The power to pi is trigged by a motion detector and will normally disappear after a timeout, then a proper shutdown is needed.
Since this happens very seldom, I’m worried that the hat never gets charged and thus, always keeping it connected to power will solve that. As far as I understand, the energy needs for the hat in standby is very low.
/C

From: Ton van Overbeek ***@***.***>
Reply to: PiSupply/PiJuice ***@***.***>
Date: Thursday, 28 October 2021 at 13:31
To: PiSupply/PiJuice ***@***.***>
Cc: Curt Lestrup ***@***.***>, Author ***@***.***>
Subject: Re: [PiSupply/PiJuice] Boot/shutdown (Discussion #790)


Do you have both the PiJuice Hat and the Pi3 connected to a power supply (= using 2 power supplies)?
If so, you need a user script regularly polling the Pi3 power input and shutdown the Pi3 when the power input disappears.
Make sure you also switch off the 5V power from the PiJuice to the Pi GPIO.
Then the Pi3 will boot when it is reconnected to external power.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/PiSupply/PiJuice/discussions/790#discussioncomment-1552115>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ACXETTSXLAVM4TUINIXYSWLUJE7C5ANCNFSM5G3KVUDA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by cumafo** [reply]
Hi,
Tried all I could think of using pijuice_cli but failed to get the pi to boot on power on, have to press hat-sw1.
How do I disable "5V power from the PiJuice to the Pi GPIO"?
Should it be done by the script?
Does it affect proper shutdown on PI power loss?
/C

**Reply by grumpyprogrammingracoon** [reply]
Hello,

Will the Pi4 start as soon as Power is provided to the PiJuice?

Or do i always have to press the SW1 Button to start my Pi4?

**Reply by tvoverbeek** [reply]
If power is only supplied to the PiJuice you will need to press SW1 to start the Pi4 (=PiJuice turns on its 5V to the GPIO)
